src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -31,11 +31,21 @@ const useStyles = makeStyles((theme) => ({
   },
   input: {
     fontSize: '1.6rem',
+    '&:hover': {
+      border: '2px solid #E5E5E5',
+    },
+    '&:focus': {
+      border: '2px solid #333E64',
+    },
     '& > *': {
       fontSize: '1.6rem !important',
       color: theme.palette.text.default,
     },
-    height: '50px',
+    height: '55px',
+    backgroundColor: '#E5E5E5',
+    paddingLeft: '10px',
+    border: '1px solid #A0A0A0',
+    borderRadius: '5px',
```

Comment:
Is there a way to use our [colour palette](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/src/theme.js)?.

See how it was done in other components, like [here](https://github.com/Seneca-CDOT/telescope/blob/4708a8718b71191ae7fe106a8f4d90fd1505b015/src/frontend/src/components/SearchBar/SearchBar.jsx#L30) or [here](https://github.com/Seneca-CDOT/telescope/blob/4708a8718b71191ae7fe106a8f4d90fd1505b015/src/frontend/src/components/SearchBar/SearchBar.jsx#L62).

Reply:
Yep I'm switching over to using the palette now.